### PR TITLE
fix(labels): add focus at create label input after click

### PR DESCRIPTION
**Pull request by SMahil**
Add focus at create label
Fix focus css for search bar in select-dropdown

### CODE REVIEW

File: src/app/components/label-selector/label-selector.component.ts [hunk 1]
```diff
@@ -139,6 +139,10 @@ export class LabelSelectorComponent implements OnInit, OnChanges {
   clickOnAddLabel() {
     this.newSelectedColor = this.colors[Math.floor(Math.random()*this.colors.length)];
     this.activeAddLabel = true;
+    setTimeout(() => {
+      this.labelnameInput.nativeElement.focus();
+      console.log("label focus");
```
Comment (vikram-raj) [inline src/app/components/label-selector/label-selector.component.ts]: Remove console log.

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2434-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2434-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2434-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2434-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2434-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2434-fabric8-planner.badger.fabric8.io

**Pull request merged**